spec/javascripts/components/story/expanded_story/expanded_story_state_spec.js

```diff
@@ -128,89 +144,95 @@ describe('<ExpandedStoryState />', () => {
     it('triggers the edit callback passing the story state', () => {
       const state = states[0];
       const change = states[1];
-
-      const story = {
-        _editing: { state },
+      const onEditSpy = vi.fn();
+      const props = {
+        story: {
+          _editing: { state },
+        },
+        onEdit: onEditSpy,
+        disabled: false,
       };
 
-      const onEdit = vi.fn();
-
-      const wrapper = shallow(
-        <ExpandedStoryState story={story} onEdit={onEdit} disabled={false} />
-      );
-      const select = wrapper.find('select');
-
-      select.simulate('change', { target: { value: change } });
+      const { container } = renderComponent(props);
+      const select = container.querySelector('select');
+      fireEvent.change(select, { target: { value: change } });
 
-      expect(onEdit).toHaveBeenCalledWith(change);
+      expect(onEditSpy).toHaveBeenCalledWith(change);
     });
   });
 
   describe('when component is disabled', () => {
     it('select field is editable', () => {
```

Comment:
Shouldn't it be enable instead editable?

Reply:
Another case of the pre-migration test just doing that, originally:

```js
describe('when the user selects a state', () => {
    it('triggers the edit callback passing the story state', () => {
      const state = states[0];
      const change = states[1];

      const story = {
        _editing: { state }
      };

      const onEdit = sinon.spy();

      const wrapper = shallow(
        <ExpandedStoryState
          story={story}
          onEdit={onEdit}
          disabled={false}
        />
      );
      const select = wrapper.find('select');

      select.simulate('change', { target: { value: change } });

      expect(onEdit).toHaveBeenCalledWith(change);
    });
  });
```

Should we change it?

Reply:
See. Then, we could replace the test title to "select field is disabled". Makes more sense I guess. What do you think?